EST.MIT.InvoiceImporter.Function/Functions/ImporterFunctions.cs

```diff
@@ -57,4 +71,21 @@ public ImporterFunctions(IAzureBlobService azureBlobService, IAzureTableService
             throw;
         }
     }
+
+    private Notification CreateNotificationRequest(ImportRequest importMessage)
+    {
+        return new NotificationBuilder()
+                                        .WithId(importMessage.ImportRequestId.ToString())
+                                        .WithScheme(importMessage.SchemeType)
+                                        .WithAction(importMessage.Status)
+                                        .WithEmailRecipient(importMessage.Email)
+                                        .WithData(new NotificationOutstandingApproval
+                                        {
+                                            Name = importMessage.Email,
+                                            Link = $"{_httpContextAccessor.HttpContext.GetBaseURI()}/invoice/details/{importMessage.SchemeType}/{importMessage.ImportRequestId}/true",
```

Comment:
Sorry, This isn't right. 
a) The ImportRequestId isn't the InvoiceId, so the link wouldn't work.
b) If any 'click' link is needed in Importer emails, it would go to the 'My Uploads' page, with no Scheme/Id necessary in the URL.

Reply:
the HttpContext isn't the right way to get the BaseURI. This is a Queue-driven function app. Look in the NotificationService (Which probably should be removed, as nothing is hitting it except tests)

var baseUrl = _configuration.GetValue<string>("WebUIBaseUrl");

The full path to the "MyImports" page should be in config too.

So the httpContext injection and related tests can be removed.